orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/poll_ingest_job.py

```diff
@@ -52,3 +52,48 @@ def check_data_ingest_job_result(config: DagsterConfigDict) -> DagsterConfigDict
         'max_wait_time_seconds': 600,  # 10 minutes
         'poll_interval_seconds': 5
     }
+
+
+@solid(
+    required_resource_keys={"data_repo_client"},
+    config_schema={
+        'max_wait_time_seconds': Int,
+        'poll_interval_seconds': Int,
+    }
+)
+def base_check_table_ingest_job_result(context: AbstractComputeExecutionContext, job_id: JobId) -> Nothing:
+    # we need to poll on the endpoint as a workaround for a race condition in TDR (DR-1791)
+    def __fetch_job_results(jid: JobId) -> Optional[JobModel]:
+        try:
+            context.log.info(f"Fetching job results for job_id = {jid}")
+            return context.resources.data_repo_client.retrieve_job_result(jid)
+        except ApiException as ae:
+            if 500 <= ae.status <= 599:
+                context.log.info(f"Data repo returned error when fetching results for job_id = {jid}, scheduling retry")
+                return None
+            raise
+
+    job_results = retry(
+        __fetch_job_results,
+        context.solid_config['max_wait_time_seconds'],
+        context.solid_config['poll_interval_seconds'],
+        is_truthy,
+        job_id
+    )
+    if not job_results:
+        raise Failure(f"No job results after polling bulk ingest, job_id = {job_id}")
+
+    if job_results['bad_row_count'] == '0':
```

Comment:
Table results return a different payload than file ingest results. I need to look into not copying and pasting here.